fastestimator/trace/trace.py

```diff
@@ -90,9 +90,15 @@ def on_epoch_begin(self, data: Data):
             self.epoch_start = time.perf_counter()
             self.step_start = time.perf_counter()
 
+    def on_batch_begin(self, data: Data):
```

Comment:
thought we were going to move this back?

Reply:
just moved it